lib/Auth/AbstractAuth.php

```diff
@@ -235,17 +234,6 @@ protected function createCurlFile($filename, $mimetype = '', $postname = '')
         return curl_file_create($filename, $mimetype, $postname);
     }
 
-    /**
-     * @param $isPost
-     * @param $parameters
-     *
-     * @return array
-     */
-    protected function getQueryParameters($isPost, $parameters)
-    {
-        return ($isPost) ? array() : $parameters;
```

Comment:
@alanhartless fixed this on the 3.0 branch like this: https://github.com/mautic/api-library/commit/44a4db754768da71cdbed0b0a1cb9297ecff310d

Reply:
Fixed :+1: